app/views/pages/privacy.erb

```diff
@@ -2,51 +2,56 @@
 
 <h1 class="govuk-heading-xl">Privacy notice: how we use your data</h1>
 
-<p>Last updated: 10 April 2024</p>
+<p>Last updated: 22 July 2024</p>
 
 <p>
-  GOV.UK Forms lets UK government teams create online forms to collect
-  information from people and organisations.
+GOV.UK Forms is a platform provided by the Government Digital Service (GDS), which is part of the Cabinet Office. The GOV.UK Forms platform lets UK government teams create online forms to collect information from people and organisations.
 </p>
 
 <p>
-  GOV.UK Forms is provided by the Government Digital Service (GDS) which is part
-  of the Cabinet Office.
+This privacy notice sets out what data we collect from you when you use the GOV.UK Forms platform, the GOV.UK Forms website (www.forms.service.gov.uk) and the services associated with them (together ‘GOV.UK Forms’). It also sets out what we do with that data.
```

Comment:
Do we want `www.forms.service.gov.uk` here to be a link?

Reply:
Shouldn't be a link - it's just defining what the privacy notice applies to